test/unit/guard.cc

```diff
@@ -11,20 +11,29 @@ sigsegf_handler(int signo)
 	exit(0);
 }
 
-static int __attribute__((noinline))
-stack_break_f(char *ptr)
+/*
+ * ASAN is disabled for this function, because for the stack-use-after-return
+ * detection it could allocate the `block' on a fake stack, rather than placing
+ * it on a fiber stack. In that case, a lot more recursive calls will be
+ * required to overflow the stack.
+ */
+static NOINLINE int
+__attribute__((no_sanitize("address")))
```

Comment:
`default_gcc_centos_7` workflow fails because of this. I guess we should use this attribute only if ASAN is enabled. It's worth adding a new macro for this to `trivia/util.h`.

Reply:
Fixed.